scripts/build.py

```diff
@@ -285,10 +361,19 @@ def get_font_filepaths(dependency_bundle, dependency_dir):
         list(str). List of paths to font files that need to be copied.
     """
     if 'fontsPath' not in dependency_bundle:
+        # Skipping dependency bundle in manifest.json that does not have
+        # fontsPath property.
         return []
     fonts_path = dependency_bundle['fontsPath']
-    return [os.path.join(dependency_dir, fonts_path, extension)
-            for extension in FONT_EXTENSIONS]
+    # Obtain directory path to /font inside dependency folder.
+    font_dir = os.path.join(dependency_dir, fonts_path)
+    # E.g. third_party/static/bootstrap-3.3.4/fonts/.
```

Comment:
This should go before the previous line; it is part of the comment that explains the previous line.

Reply:
Done